app/views/reports/_new_report_item_form.html.erb

```diff
@@ -1,6 +1,10 @@
 <%= form_for [@report, @report_item], remote: true, id: 'report_item_form' do |f| %>
   <%= f.error_messages %>
-  <%= f.text_area :content, placeholder: 'Ctrl+Enter to quickly add to report!', id: 'item', size: '40x3', wrap: 'hard', class: 'form-control' %>
+  <% if request.env['HTTP_USER_AGENT'].downcase.match(/mac/i) %>
+    <%= f.text_area :content, placeholder: 'Cmd+Enter to quickly add to report!', id: 'item', size: '40x3', wrap: 'hard', class: 'form-control' %>
+  <% else %>
+    <%= f.text_area :content, placeholder: 'Ctrl+Enter to quickly add to report!', id: 'item', size: '40x3', wrap: 'hard', class: 'form-control' %>
+  <% end %>
   <%= f.submit "Add to report", id: 'report_item_submit', title: 'Ctrl+Enter to add to shift report', class: 'btn btn-primary', data: { disable_with: "Submitting..." } %>
```

Comment:
Actually probably a better way to implement this would be to write a Helper method that takes care of the evaluation. Like:

```ruby
def report_kbd_shortcut
  if request.env['HTTP_USER_AGENT'].downcase.match(/mac/i)
    'Cmd+Enter to quickly add to report!'
  else
    'Ctrl+Enter to quickly add to report!'
  end
end
```
And then call that method both on the placeholder and on the title attribute here.
And then use that